code/__DEFINES/machines.dm

```diff
@@ -27,6 +27,9 @@
 #define IDLE_POWER_USE 1
 #define ACTIVE_POWER_USE 2
 
+#define MIN_IDLE_POWER_USAGE 100
+#define MIN_ACTIVE_POWER_USAGE 1000
```

Comment:
This isn't the most descriptive name

The main goal here is to give people setting power usage an anchor point for their choice of power usage.
It might be useful? to base this off some standard number, something people are likely to have some experience with ingame. Not totally sure what to use, but i think it'd help some

Reply:
thing is, there aren't standard numbers when it comes to power, i'm trying to standardize the amounts of 100 W for baseline idle power consumption on most machines and 1 kW for baseline active power consumption. I don't know how more descriptive or more baseline i can make this.
I think that clear and round numbers like 100 and 1000 are enough to make people understand what the baseline is and what is high amount and low amount.
For the names if you have suggestions i'm open for it

Reply:
Lemme think about it.
If they are the baseline that's ok, but I really do think you want some sort of VENT_USE or something concrete people can hold in their heads. That's maybe a bad example, but you get me

Reply:
it's a base machine stat, i don't know how to improve the name other than having literally base machine idle consumption

Reply:
mmf